osgp/platform/osgp-secret-management/pom.xml

```diff
@@ -0,0 +1,192 @@
+    <artifactId>parent-platform</artifactId>
+    <version>5.2.0-SNAPSHOT</version>
+    <relativePath>../parent-platform/pom.xml</relativePath>
+  </parent>
+
+  <properties>
+    <java.version>1.8</java.version>
+    <spring.boot.version>2.2.2.RELEASE</spring.boot.version>
+    <jaxb.plugin.version>2.5.0</jaxb.plugin.version>
+    <build.helper.maven.plugin.version>3.0.0</build.helper.maven.plugin.version>
+    <apache.ws.xlmschema.version>2.0.1</apache.ws.xlmschema.version>
+    <apache.jms.api.version>2.0.1</apache.jms.api.version>
+    <logback.spring.extensions.version>0.1.5</logback.spring.extensions.version>
+    <wsdl4j.version>1.6.3</wsdl4j.version>
+    <maven.failsafe.plugin.version>3.0.0-M4</maven.failsafe.plugin.version>
+    <spring.boot.maven.plugin.version>2.2.2.RELEASE</spring.boot.maven.plugin.version>
+    <osgp-ws-secret-management-version>${project.parent.version}</osgp-ws-secret-management-version>
+    <com.h2database>1.4.200</com.h2database>
+    <failOnMissingWebXml>false</failOnMissingWebXml>
+  </properties>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-maven-plugin</artifactId>
+        <version>${spring.boot.maven.plugin.version}</version>
+      </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>build-helper-maven-plugin</artifactId>
+        <version>${build.helper.maven.plugin.version}</version>
+        <executions>
+          <execution>
+            <id>add-integration-test-source</id>
+            <phase>generate-test-sources</phase>
+            <goals>
+              <goal>add-test-source</goal>
+            </goals>
+            <configuration>
+              <sources>
+                <source>src/integration-test/java</source>
+              </sources>
+            </configuration>
+          </execution>
+          <execution>
+            <id>add-test-resource</id>
+            <phase>generate-test-sources</phase>
+            <goals>
+              <goal>add-test-resource</goal>
+            </goals>
+            <configuration>
+              <resources>
+                <resource>
+                  <directory>src/integration-test/resources</directory>
+                </resource>
+              </resources>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+
+      <plugin>
+        <artifactId>maven-failsafe-plugin</artifactId>
+        <version>${maven.failsafe.plugin.version}</version>
+        <executions>
+          <execution>
+            <goals>
+              <goal>integration-test</goal>
+              <goal>verify</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <version>${maven.surefire.plugin.version}</version>
+      </plugin>
+    </plugins>
+  </build>
+
+  <dependencies>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this line start at column 3. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=xml%3AIndentCheck)